src/coreclr/src/jit/lower.cpp

```diff
@@ -6278,6 +6278,7 @@ void Lowering::ContainCheckNode(GenTree* node)
             break;
         case GT_STOREIND:
             ContainCheckStoreIndir(node->AsIndir());
+            __fallthrough; // TODO: is this actually a bug?
```

Comment:
Please change it to `break`, it does not lead to an incorrect execution, but `ContainCheckStoreIndir` calls `ContainCheckIndir` itself and we don't need to call it again (but it does not hurt correctness).